.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "access": "public",
+  "baseBranch": "master",
+  "bumpVersionsWithWorkspaceProtocolOnly": true,
+  "changelog": [
+    "@changesets/changelog-github",
+    {
+      "repo": "solana-labs/solana-web3.js"
+    }
+  ],
+  "commit": false,
+  "fixed": [["@solana/!({*-impl,build-scripts,crypto-impl,test-*,tsconfig,web3.js-legacy-sham})"]],
```

Comment:
What we're saying here is ‘yeah, bump the version of everything, but leave the internal packages alone.’

Reply:
> Is legacy-sham supposed to be internal?

Felt cute, might delete later. I'm no longer convinced it's going to be useful for any practical purpose.

Reply:
> `crypto-impl` matches the first glob pattern

👀